llama-index-core/llama_index/core/callbacks/token_counting.py

```diff
@@ -30,6 +30,7 @@ class TokenCountingEvent:
     completion_token_count: int
     prompt_token_count: int
     total_token_count: int = 0
+    cached_tokens: int = 0 
```

Comment:
this works for openai. But for llms that charge based on cache_read and cache_write (like anthropic), this approach doesn't translate well 🤔